src/Logger-Arduino_AllPins/Logger-Arduino_AllPins.ino

```diff
@@ -5,17 +5,10 @@ String EOL;
 
 void setup() {
 
-   for (int i=2; i <= 13; i++){
-  
+   for (int i=0; i <= NUM_DIGITAL_PINS; i++){
     pinMode(i, INPUT);
    }
```

Comment:
That's still inconsistent. Which are the pin numbers that need to be set to INPUT?

Reply:
I mean, on one hand there is the channel, like D0, digital-channel zero. But it is not on Pin 0 of the arduino (or at least it would be a great coincidence)

Reply:
Hm, forget the previous comment. I think arduino automatically maps the digital pin number to the correct physical port. But then I don't understand how the analog read can work with the driver, because it seems to work differently for the analog pins (which are referenced by name).

Another issue: NUM_DIGITAL_PINS includes the analog pins, which can be used in digital mode. In that case, all analog pins are configured as digital inputs and can no longer be used as analog ones?

Reply:
Analog pins can be used as general-purpose I/O pins as well. In case a pin has been set to output before, the pins must be set to input for the AnalogRead call to work. There is no extra command to set the Inpute-mode specifically to Analog Input.
https://docs.arduino.cc/learn/microcontrollers/analog-input/#details-and-caveats